rpc_util.go

```diff
@@ -522,6 +522,42 @@ func (o MaxRetryRPCBufferSizeCallOption) before(c *callInfo) error {
 }
 func (o MaxRetryRPCBufferSizeCallOption) after(c *callInfo, attempt *csAttempt) {}
 
+// PerRPCUnaryClientInterceptor returns a CallOption that specifies an interceptor
+// for unary RPCs. Interceptors are invoked in order they are passed to unary RPCs.
+//
+// CallOption interceptors are invoked after DialOption interceptors.
+func PerRPCUnaryClientInterceptor(f UnaryClientInterceptor) CallOption {
```

Comment:
Let's mark this as experimental for now to make sure the semantics work well for our users before we declare it is stable.